code/player/TTTPlayer.RPCs.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace TTTReborn.Player
 {
+    using System;
```

Comment:
Okay last thing: What is this? :D You already had this multiple times now, is there any reason for this?

Reply:
the IDE automatically imports stuff there for me, if you see it lmk.